### PR TITLE
Add notes improvements

### LINKED ISSUE #85: Improvements for Add, Delete notes v2.0

### PR DESCRIPTION
**Pull request by Nazryl**
Closes #85

### CODE REVIEW

File: src/main/java/seedu/duke/ui/Formatter.java [hunk 1]
```diff
@@ -77,13 +85,10 @@ public static String formatNotes(String header, ArrayList<Note> notes) {
     public static String formatNote(String header, Note note) {
         String formattedString = "";
 
-        if (!note.getTagsName().isBlank()) {
-            header = header.concat(" " + note.getTagsName());
-        }
+        header = header.concat(" " + note.getTagsName());
```
Comment (Chongjx) [inline src/main/java/seedu/duke/ui/Formatter.java:88]: Realise that this + note.getTagsName can be added when this function is called. i.e. in the command execute

return formatNote(header + note.getTitle + note.getTagsName(), note)

**Review by Chongjx — APPROVED** [review]
Looks good! Just 1 minor thing which I thought can be adjusted

File: src/main/java/seedu/duke/ui/Formatter.java [hunk 2]
```diff
@@ -253,10 +258,10 @@ private static String encloseRow(String message) {
      * Returns the number of ascii code that appear within the message. Also match the start and end of index of each
      * color.
      *
-     * @param message String of the message to check.
+     * @param message                     String of the message to check.
      * @param coloredStringStartIndexList Arraylist of index to mark the start of a color.
-     * @param coloredStringEndIndexList Arraylist of index to mark the end of a color.
-     * @param stringColorList Arraylist of string to store the color present.
+     * @param coloredStringEndIndexList   Arraylist of index to mark the end of a color.
+     * @param stringColorList             Arraylist of string to store the color present.
```
Comment (R-Ramana) [inline src/main/java/seedu/duke/ui/Formatter.java:264]: All of our Javadocs was @param parameter definition. There was no additional whitespace to align the columns. If we are doing this, should we then standardize for all JavaDocs? Would it be easier to just have a single space?

**Review by R-Ramana — APPROVED** [review]

File: src/main/java/seedu/duke/ui/Formatter.java (hunk 2 above)
Comment (Nazryl) [reply src/main/java/seedu/duke/ui/Formatter.java:264]: I didnt realise this. I followed the format from addressbook to make it tidy and easy on the eyes. Jx and I used whitespaces when implementing the classes in the first place. I think its a minor matter as long its readable.

File: src/main/java/seedu/duke/ui/Formatter.java (hunk 1 above)
Comment (Nazryl) [reply src/main/java/seedu/duke/ui/Formatter.java:88]: wouldnt it makes sense to break all of the note stuff in the formatter? and just pass the success parameter?

**Pull request merged**